episodes/image-display.md

```diff
@@ -0,0 +1,358 @@
+- What factors could help you decide if the plugin is well maintained?
+- What factors could help you decide if the plugin is popular with Napari users?
+
+:::::::::::::::::::::::: solution 
+
+## Solution
+
+### Is a plugin well maintained?
+
+Some factors to look for:
+
+**Last updated**  
+Check when the plugin was last updated - was it recently? This is shown in the 
+search list summary and under 'Maintenance' in the activity tab. In the activity 
+tab, you can also look at a graph summarising the 'commits' over the past year - 
+'commits' are made when someone updates their plugin with new changes.
+
+**Documentation**  
+Is the plugin summary (+ any linked documentation) detailed enough to explain 
+how to use the plugin?
+
+### Is a plugin popular?
+
+Some factors to look for:
+
+**Installs**  
+Check how many times a plugin has been installed - a higher number of installs 
+usually means it's more popular in the Napari community. The installs are shown 
+in the search list summary and under 'Usage' in the activity tab. In the 
+activity tab, you can also look at a graph summarising the installs over the 
+past year.
+
+**Image.sc**  
+It can also be useful to search the plugin's name on the 
+[image.sc](https://forum.image.sc/) forum to see if other people had good 
+experiences with it.
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+## Image histograms
+
+Let's use our newly installed plugin to look at the human mitosis image. If you 
+don't have it open, go the top menubar and select:
+`File > Open Sample > napari builtins > Human Mitosis`
+
+Then open the image histogram with:  
+`Plugins > napari Matplotlib > Histogram`
+
+You should see a histogram open on the right side of the image:
+
+![](fig/mitosis-histogram.png
+){alt="Screenshot of image histogram for mitosis image" width='70%'}
+
+Here, I've changed Napari to light mode to make the histogram easier to see 
+(`File > Preferences > Appearance > Theme > light`), but you're welcome to 
+continue using dark mode if you prefer.
+
+This histogram summarises the pixel values of the entire image. On the x axis 
+is the pixel value which run from 0-255 for this 8-bit image. This is split 
+into a number of 'bins' of a certain width (for example, it could be 0-10, 11-20 
+and so on...). Each bin has a blue bar whose height is proprotional to the 
+number of pixels with values in that bin. So, for example, for our mitosis image 
+we see the highest bars to the left, with shorter bars to the right. This means 
+this image has a lot of very dark (low intensity) pixels and fewer bright 
+(high intensity) pixels.
+
+Let's quickly compare to another image. Open the 'coins' image with:  
+`File > Open Sample > napari builtins > Coins`
+
+![](fig/coins-histogram.png
+){alt="Screenshot of image histogram for coins image" width='68%'}
+
+From the histogram, we can see that this image has a wider spread of pixel 
```

Comment:
As mentioned with the picture, the histogram here looks odd and it might be worth explaining those funny spikes.